.msync.yml

```diff
@@ -1 +1 @@
-modulesync_config_version: '1.1.0'
+modulesync_config_version: '1.2.1'
```

Comment:
The commit and the PR title should be 'modulesync [version]', and, as was pointed out to me the other day, you should check out the latest tag (1.2.0 in this case), and run the modulesync off of that.

Reply:
Sure thing.  Any idea why we have to checkout a tag?

Reply:
Made the appropriate changes.  I'm assuming i updated this a couple a weeks ago so this is probably a downgrade as far as msync goes.

Reply:
Yeah, I thought the 1.2.0 tag got created, but it didn't.
I think basing it on the tag is just to ensure that the versioning is consistent (that is, if it says 1.2.0, it's 1.2.0 exactly and not 1.2.0++.

https://github.com/voxpupuli/modulesync_config#how-to-use-it